### ISSUE TITLE
Does this project have a logo?

### ISSUE THREAD

**Issue by edwardloveall** [samphippen/rubyfmt#375]
_This issue isn't a bug report so if you feel like it's not appropriate here, feel free to close!_

I am working on an editor extension for [Nova](https://nova.app/) and their extensions require an icon. I'm happy to adapt an icon from a logo but I didn't see one. Thanks!

**Comment by reese** [comment]
I don't believe there's anything currently, no. We've been looking for someone to design one though (and if anyone else reads this is interested, let us know!), but logos for formatters are tough!

**Comment by fables-tales** [comment]
I believe @baweaver is working on a logo

**Comment by edwardloveall** [comment]
Thanks! I made a super bad one [here](https://github.com/edwardloveall/rubyfmt.novaextension) and I am excited for a proper one 😄
